examples/sample_measurement/pyproject.toml

```diff
@@ -14,6 +14,7 @@ mypy = ">=1.0"
 grpc-stubs = "^1.53"
 # Uncomment to use prerelease dependencies.
 # ni-measurementlink-service = {path = "../..", develop = true}
+types-protobuf = "^4.21"
```

Comment:
[style] Let's keep the type stub modules together

Reply:
Moved above the commented out dependencies as suggested.